docs/platforms/javascript/common/user-feedback/v7/index.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: Configuration (deprecated)
+sidebar_order: 6200
+description: "Learn about the general User Feedback configuration fields for version 7 of the JavaScript SDK."
+---
+
+<PlatformSection notSupported={["javascript.node", "javascript.aws-lambda", "javascript.azure-functions", "javascript.connect", "javascript.express", "javascript.fastify", "javascript.gcp-functions", "javascript.koa"]}>
+
+## User Feedback Widget
+
+The User Feedback Widget offers many customization options, and if the available options are insufficient, you can [use your own UI](#bring-your-own-widget). The following image shows which elements are customizable. The configuration keys on the left side of the image correspond to [text customization](#text-customization) and the configuration keys on the right side of the image are for [theme customizations](#theme-customization).
```

Comment:
todo: we should mention that this version is deprecated (was beta, and v8 is the GA) and migration docs are available at https://github.com/getsentry/sentry-javascript/blob/develop/docs/migration/feedback.md